build-system/compile/bundles.config.js

```diff
@@ -1106,6 +1106,12 @@ exports.extensionBundles = [
     options: {hasCss: true},
     type: TYPES.MISC,
   },
+  {
+    name: 'amp-vi-player',
```

Comment:
Thanks for contributing to AMP! We recently introduced a naming guide for 3p components (https://github.com/ampproject/amphtml/blob/master/spec/amp-3p-naming.md). Does Video Intelligence have a copyright on the vi name? We want to ensure that there is no confusion or duplication down the road with other 3p vendors.